packages/asset-swapper/src/utils/market_operation_utils/aave_reserves_cache.ts

```diff
@@ -0,0 +1,115 @@
+import { logUtils } from '@0x/utils';
+import { gql, request } from 'graphql-request';
+
+import { constants } from '../../constants';
+
+const RESERVES_GQL_QUERY = gql`
+    {
+        reserves(where: { isActive: true, isFrozen: false }, orderBy: totalLiquidity, orderDirection: desc) {
+            id
+            name
+            isActive
+            isFrozen
+            underlyingAsset
+            totalLiquidity
+            aToken {
+                id
+                underlyingAssetAddress
```

Comment:
Is there a difference between this and the outer `underlyingAsset`?

Reply:
Nope 🙈